homeassistant/components/backup/manager.py

```diff
@@ -275,12 +282,25 @@ async def _async_upload_backup(
         finally:
             self.syncing = False
 
-    async def async_get_backups(self, **kwargs: Any) -> dict[str, Backup]:
+    async def async_get_backups(
+        self, **kwargs: Any
+    ) -> tuple[dict[str, Backup], dict[str, Exception]]:
         """Return backups."""
         backups: dict[str, Backup] = {}
-        for agent_id, agent in self.backup_agents.items():
-            agent_backups = await agent.async_list_backups()
-            for agent_backup in agent_backups:
+        agent_errors: dict[str, Exception] = {}
+        agent_ids = list(self.backup_agents.keys())
```

Comment:
```suggestion
        agent_ids = list(self.backup_agents)
```

Reply:
Fixed in https://github.com/home-assistant/core/pull/130890